src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ItemFilter.cs

```diff
@@ -5,6 +5,9 @@ namespace NuGet.VisualStudio.Internal.Contracts
 {
     public enum ItemFilter
     {
+        /// <summary>
+        /// Represents Browse tab in PM UI
+        /// </summary>
```

Comment:
As per Andy's comment in https://github.com/NuGet/Client.Engineering/issues/2222#issuecomment-1497100858, this entire project can be removed but that is out of scope for this PR. I added a comment here to avoid confusion.